app/src/main/java/org/fossasia/phimpme/editor/editimage/EditImageActivity.java

```diff
@@ -81,13 +83,13 @@
     protected int mOpTimes = 0;
     protected boolean isBeenSaved = false;
 
-    LoadImageTask mLoadImageTask;
+    private LoadImageTask mLoadImageTask;
 
     private EditImageActivity mContext;
     public Bitmap mainBitmap;
     private Bitmap originalBitmap;
     public ImageViewTouch mainImage;
-    private View cancel,save,bef_aft,undo;
+    private View cancel,save,bef_aft,undo,redo;
```

Comment:
@vinaysajjanapu Why aren't you using ButterKnife for all the findVIewById calls?
Please update this PR and use butterknife for all the view lookups in this java class.